hack/kubevirt-builder/Dockerfile.ci

```diff
@@ -0,0 +1,131 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+
+# Install packages
+# dnf copr enable -y @virtmaint-sig/virt-preview with yum?
+# python3-pip-9.0.3-5.el7.noarch \
+RUN yum install -y yum-plugin-copr && \
+    yum copr enable -y vbatts/bazel epel-7 && \
+    yum -y install \
+        libvirt-devel \
+        bazel \
+        cpio \
+        patch \
+        make \
+        git \
+        mercurial \
+        sudo \
+        gcc \
+        gcc-c++ \
+        glibc-devel \
+        findutils \
+        qemu-img \
+        protobuf-compiler \
+        python3-devel \
+        python3-pip \
+        redhat-rpm-config \
+        jq \
+        unzip && \
+    yum clean all && rm -rf /var/cache/yum/*
+
+# install gradle
+RUN yum install -y java-1.8.0-openjdk-devel && \
```

Comment:
what do we need gradle for?

Reply:
I dunno, it was in the build Dockerfile so I included it here (but have to install like so in centos)  I'll remove it and test, see if we need it

Reply:
I've confirmed that gradle is not necessary, I've removed it